contracts/src/v0.8/functions/dev/v1_X/ocr/OCR2Base.sol

```diff
@@ -320,8 +316,10 @@ abstract contract OCR2Base is ConfirmedOwner, OCR2Abstract {
       emit Transmitted(configDigest, uint32(epochAndRound >> 8));
 
       ConfigInfo memory configInfo = s_configInfo;
```

Comment:
Do we need this line?

Reply:
It is used when determining the minimum signers, but I checked it with and without, it's cheaper gas-wise without.